[icu] Build error on x64-windows

Package: icu[core,tools]:x64-windows@74.2#5

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.44.35213.0
-    vcpkg-tool version: 2025-04-01-9c604254140797833b6f76908435c9fcbf09920e
    vcpkg-readonly: true
    vcpkg-scripts version: 4f8fe05871555c1798dbcb1957d0d595e94f7b57

**To Reproduce**

`vcpkg install `

**Failure logs**

```
-- Using cached icu4c-74_2-src.tgz
-- Extracting source C:/Users/Edward/AppData/Local/vcpkg/downloads/icu4c-74_2-src.tgz
-- Applying patch disable-escapestr-tool.patch
-- Applying patch remove-MD-from-configure.patch
-- Applying patch fix_parallel_build_on_windows.patch
-- Applying patch fix-extra.patch
-- Applying patch mingw-dll-install.patch
-- Applying patch disable-static-prefix.patch
-- Applying patch fix-win-build.patch
-- Applying patch vcpkg-cross-data.patch
-- Applying patch darwin-rpath.patch
-- Applying patch mingw-strict-ansi.diff
-- Applying patch cleanup_msvc.patch
-- Using source at E:/kicad-master/build/vcpkg_installed/vcpkg/blds/icu/src/c-74_2-src-2c60f3f6bd.clean
-- Found external ninja('1.12.1').
-- Getting CMake variables for x64-windows
-- Using cached msys2-autoconf-wrapper-20240607-1-any.pkg.tar.zst
-- Using cached msys2-automake-wrapper-20240607-1-any.pkg.tar.zst
-- Using cached msys2-autoconf-archive-2023.02.20-1-any.pkg.tar.zst
-- Using cached msys2-binutils-2.43.1-1-x86_64.pkg.tar.zst
-- Using cached msys2-libtool-2.4.7-4-x86_64.pkg.tar.zst
-- Using cached msys2-make-4.4.1-2-x86_64.pkg.tar.zst
-- Using cached msys2-which-2.21-4-x86_64.pkg.tar.zst
-- Using cached msys2-bash-5.2.037-1-x86_64.pkg.tar.zst
-- Using cached msys2-coreutils-8.32-5-x86_64.pkg.tar.zst
-- Using cached msys2-file-5.45-3-x86_64.pkg.tar.zst
-- Using cached msys2-gawk-5.3.1-1-x86_64.pkg.tar.zst
-- Using cached msys2-grep-1~3.0-6-x86_64.pkg.tar.zst
-- Using cached msys2-gzip-1.13-1-x86_64.pkg.tar.zst
-- Using cached msys2-diffutils-3.10-1-x86_64.pkg.tar.zst
-- Using cached msys2-pkgconf-2.3.0-1-x86_64.pkg.tar.zst
-- Using cached msys2-sed-4.9-1-x86_64.pkg.tar.zst
-- Using cached msys2-msys2-runtime-3.5.4-2-x86_64.pkg.tar.zst
-- Using cached msys2-autoconf2.72-2.72-1-any.pkg.tar.zst
-- Using cached msys2-automake1.16-1.16.5-1-any.pkg.tar.zst
-- Using cached msys2-automake1.17-1.17-1-any.pkg.tar.zst
-- Using cached msys2-libiconv-1.17-1-x86_64.pkg.tar.zst
-- Using cached msys2-libintl-0.22.4-1-x86_64.pkg.tar.zst
-- Using cached msys2-zlib-1.3.1-1-x86_64.pkg.tar.zst
-- Using cached msys2-tar-1.35-2-x86_64.pkg.tar.zst
-- Using cached msys2-findutils-4.10.0-2-x86_64.pkg.tar.zst
-- Using cached msys2-gmp-6.3.0-1-x86_64.pkg.tar.zst
-- Using cached msys2-gcc-libs-13.3.0-1-x86_64.pkg.tar.zst
-- Using cached msys2-libbz2-1.0.8-4-x86_64.pkg.tar.zst
-- Using cached msys2-liblzma-5.6.3-1-x86_64.pkg.tar.zst
-- Using cached msys2-libzstd-1.5.6-1-x86_64.pkg.tar.zst
-- Using cached msys2-libreadline-8.2.013-1-x86_64.pkg.tar.zst
-- Using cached msys2-mpfr-4.2.1-1-x86_64.pkg.tar.zst
-- Using cached msys2-libpcre-8.45-4-x86_64.pkg.tar.zst
-- Using cached msys2-m4-1.4.19-2-x86_64.pkg.tar.zst
-- Using cached msys2-perl-5.38.2-2-x86_64.pkg.tar.zst
-- Using cached msys2-ncurses-6.5.20240831-2-x86_64.pkg.tar.zst
-- Using cached msys2-libxcrypt-4.4.36-1-x86_64.pkg.tar.zst
-- Using msys root at C:/Users/Edward/AppData/Local/vcpkg/downloads/tools/msys2/d1518c5fb772aeb9
-- Generating configure for x64-windows
-- Finished generating configure for x64-windows
-- Using cached msys2-mingw-w64-x86_64-pkgconf-1~2.3.0-1-any.pkg.tar.zst
-- Using cached msys2-msys2-runtime-3.5.4-2-x86_64.pkg.tar.zst
-- Using msys root at C:/Users/Edward/AppData/Local/vcpkg/downloads/tools/msys2/21caed2f81ec917b
-- Configuring x64-windows-dbg
-- Configuring x64-windows-rel
-- Using cached msys2-autoconf-wrapper-20240607-1-any.pkg.tar.zst
-- Using cached msys2-automake-wrapper-20240607-1-any.pkg.tar.zst
-- Using cached msys2-autoconf-archive-2023.02.20-1-any.pkg.tar.zst
-- Using cached msys2-binutils-2.43.1-1-x86_64.pkg.tar.zst
-- Using cached msys2-libtool-2.4.7-4-x86_64.pkg.tar.zst
-- Using cached msys2-make-4.4.1-2-x86_64.pkg.tar.zst
-- Using cached msys2-which-2.21-4-x86_64.pkg.tar.zst
-- Using cached msys2-bash-5.2.037-1-x86_64.pkg.tar.zst
-- Using cached msys2-coreutils-8.32-5-x86_64.pkg.tar.zst
-- Using cached msys2-file-5.45-3-x86_64.pkg.tar.zst
-- Using cached msys2-gawk-5.3.1-1-x86_64.pkg.tar.zst
-- Using cached msys2-grep-1~3.0-6-x86_64.pkg.tar.zst
-- Using cached msys2-gzip-1.13-1-x86_64.pkg.tar.zst
-- Using cached msys2-diffutils-3.10-1-x86_64.pkg.tar.zst
-- Using cached msys2-pkgconf-2.3.0-1-x86_64.pkg.tar.zst
-- Using cached msys2-sed-4.9-1-x86_64.pkg.tar.zst
-- Using cached msys2-msys2-runtime-3.5.4-2-x86_64.pkg.tar.zst
-- Using cached msys2-autoconf2.72-2.72-1-any.pkg.tar.zst
-- Using cached msys2-automake1.16-1.16.5-1-any.pkg.tar.zst
-- Using cached msys2-automake1.17-1.17-1-any.pkg.tar.zst
-- Using cached msys2-libiconv-1.17-1-x86_64.pkg.tar.zst
-- Using cached msys2-libintl-0.22.4-1-x86_64.pkg.tar.zst
-- Using cached msys2-zlib-1.3.1-1-x86_64.pkg.tar.zst
-- Using cached msys2-tar-1.35-2-x86_64.pkg.tar.zst
-- Using cached msys2-findutils-4.10.0-2-x86_64.pkg.tar.zst
-- Using cached msys2-gmp-6.3.0-1-x86_64.pkg.tar.zst
-- Using cached msys2-gcc-libs-13.3.0-1-x86_64.pkg.tar.zst
-- Using cached msys2-libbz2-1.0.8-4-x86_64.pkg.tar.zst
-- Using cached msys2-liblzma-5.6.3-1-x86_64.pkg.tar.zst
-- Using cached msys2-libzstd-1.5.6-1-x86_64.pkg.tar.zst
-- Using cached msys2-libreadline-8.2.013-1-x86_64.pkg.tar.zst
-- Using cached msys2-mpfr-4.2.1-1-x86_64.pkg.tar.zst
-- Using cached msys2-libpcre-8.45-4-x86_64.pkg.tar.zst
-- Using cached msys2-m4-1.4.19-2-x86_64.pkg.tar.zst
-- Using cached msys2-perl-5.38.2-2-x86_64.pkg.tar.zst
-- Using cached msys2-ncurses-6.5.20240831-2-x86_64.pkg.tar.zst
-- Using cached msys2-libxcrypt-4.4.36-1-x86_64.pkg.tar.zst
-- Using msys root at C:/Users/Edward/AppData/Local/vcpkg/downloads/tools/msys2/d1518c5fb772aeb9
-- Building/Installing x64-windows-dbg
-- Making target 'all' for x64-windows-dbg
-- Making target 'install' for x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: C:/Users/Edward/AppData/Local/vcpkg/downloads/tools/msys2/d1518c5fb772aeb9/usr/bin/bash.exe --noprofile --norc --debug -c "CPP='compile cl.exe -E' CC='compile cl.exe' CXX='compile cl.exe' CC_FOR_BUILD='compile cl.exe' CPP_FOR_BUILD='compile cl.exe -E' CXX_FOR_BUILD='compile cl.exe' RC='windres-rc rc.exe' WINDRES='windres-rc rc.exe' AR='ar-lib lib.exe' LD='link.exe -verbose' RANLIB=':' STRIP=': STRIP-disabled' NM='dumpbin.exe -symbols -headers' DLLTOOL='link.exe -verbose -dll' CCAS=': CCAS-disabled' AS=': AS-disabled' C:/Users/Edward/AppData/Local/vcpkg/downloads/tools/msys2/d1518c5fb772aeb9/usr/bin/make.exe V=1 -j 9 --trace -f Makefile install DESTDIR=/E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows"
    Working Directory: E:/kicad-master/build/vcpkg_installed/vcpkg/blds/icu/x64-windows-dbg
    See logs for more information:
      E:\kicad-master\build\vcpkg_installed\vcpkg\blds\icu\make-install-x64-windows-dbg-out.log
      E:\kicad-master\build\vcpkg_installed\vcpkg\blds\icu\make-install-x64-windows-dbg-err.log

Call Stack (most recent call first):
  E:/kicad-master/build/vcpkg_installed/x64-windows/share/vcpkg-make/vcpkg_make.cmake:68 (vcpkg_execute_build_process)
  E:/kicad-master/build/vcpkg_installed/x64-windows/share/vcpkg-make/vcpkg_make_install.cmake:106 (vcpkg_run_shell_as_build)
  C:/Users/Edward/AppData/Local/vcpkg/registries/git-trees/91b03be43850140e64dba01dcadc8b3f73b1083b/portfile.cmake:83 (vcpkg_make_install)
  scripts/ports.cmake:203 (include)



```

<details><summary>E:\kicad-master\build\vcpkg_installed\vcpkg\blds\icu\make-install-x64-windows-dbg-err.log</summary>

```
make[1]: ./../src/c-74_2-src-2c60f3f6bd.clean/source/install-sh: No such file or directory
make[1]: *** [Makefile:87: install-library] Error 127
make: *** [Makefile:153: install-recursive] Error 2
make: *** Waiting for unfinished jobs....
```
</details>

<details><summary>E:\kicad-master\build\vcpkg_installed\vcpkg\blds\icu\make-install-x64-windows-dbg-out.log</summary>

```
Makefile:153: update target 'install-recursive' due to: target is .PHONY
dot_seen=no; \
target=`echo install-recursive | sed s/-recursive//`; \
list='stubdata common i18n  io tools data extra  '; for subdir in $list; do \
  echo "C:/Users/Edward/AppData/Local/vcpkg/downloads/tools/msys2/d1518c5fb772aeb9/usr/bin/make[0]: Making \`$target' in \`$subdir'"; \
  if test "$subdir" = "."; then \
    dot_seen=yes; \
    local_target="$target-local"; \
  else \
    local_target="$target"; \
  fi; \
  (cd $subdir && C:/Users/Edward/AppData/Local/vcpkg/downloads/tools/msys2/d1518c5fb772aeb9/usr/bin/make RECURSIVE=YES $local_target) || exit; \
done; \
if test "$dot_seen" = "no"; then \
  C:/Users/Edward/AppData/Local/vcpkg/downloads/tools/msys2/d1518c5fb772aeb9/usr/bin/make "$target-local" || exit; \
fi
Makefile:184: update target 'install-icu' due to: target is .PHONY
/bin/sh ./../src/c-74_2-src-2c60f3f6bd.clean/source/mkinstalldirs /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/share/icu/icud/74.2/config
Makefile:364: update target 'config/icu-config.1' due to: target does not exist
cd . \
 && CONFIG_FILES=./config/icu-config.1 CONFIG_HEADERS= /bin/sh ./config.status
C:/Users/Edward/AppData/Local/vcpkg/downloads/tools/msys2/d1518c5fb772aeb9/usr/bin/make[0]: Making `install' in `stubdata'
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E
make[1]: Entering directory '/e/kicad-master/build/vcpkg_installed/vcpkg/blds/icu/x64-windows-dbg/stubdata'
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build
.././../src/c-74_2-src-2c60f3f6bd.clean/source/config/mh-msys-msvc:208: update target 'icudtd.dll' due to: target does not exist
echo -n
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows
([ -e icudtd74.dll.manifest ] && ( echo Embedding manifest into icudtd74.dll && mt.exe -nologo -manifest icudtd74.dll.manifest -outputresource:"icudtd74.dll;2" && rm -rf icudtd74.dll.manifest )) || true
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/share
Makefile:86: update target 'install-library' due to: target is .PHONY
/bin/sh .././../src/c-74_2-src-2c60f3f6bd.clean/source/mkinstalldirs /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/lib
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/share/icu
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/share/icu/icud
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/share/icu/icud/74.2
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/share/icu/icud/74.2/config
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/lib
/bin/sh ./../src/c-74_2-src-2c60f3f6bd.clean/source/mkinstalldirs /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/lib/icud/74.2
./../src/c-74_2-src-2c60f3f6bd.clean/source/install-sh -c icudtd74.dll /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/lib
make[1]: Leaving directory '/e/kicad-master/build/vcpkg_installed/vcpkg/blds/icu/x64-windows-dbg/stubdata'
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/lib/icud
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/lib/icud/74.2
/bin/sh ./../src/c-74_2-src-2c60f3f6bd.clean/source/mkinstalldirs /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/../tools/icu/debug/bin
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/../tools
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/../tools/icu
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/../tools/icu/debug
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/../tools/icu/debug/bin
/bin/sh ./../src/c-74_2-src-2c60f3f6bd.clean/source/mkinstalldirs /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/../tools/icu/debug/bin
./../src/c-74_2-src-2c60f3f6bd.clean/source/install-sh -c -m 644 ./../src/c-74_2-src-2c60f3f6bd.clean/source/config/mh-msys-msvc /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/share/icu/icud/74.2/config/mh-msys-msvc
./../src/c-74_2-src-2c60f3f6bd.clean/source/install-sh -c ./../src/c-74_2-src-2c60f3f6bd.clean/source/mkinstalldirs /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/share/icu/icud/74.2/mkinstalldirs
config.status: creating ./config/icu-config.1
./../src/c-74_2-src-2c60f3f6bd.clean/source/install-sh -c ./../src/c-74_2-src-2c60f3f6bd.clean/source/install-sh /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/share/icu/icud/74.2/install-sh
/bin/sh ./../src/c-74_2-src-2c60f3f6bd.clean/source/mkinstalldirs /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/lib/pkgconfig
mkdir /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/lib/pkgconfig
./../src/c-74_2-src-2c60f3f6bd.clean/source/install-sh -c -m 644 ./config/icu-uc.pc ./config/icu-i18n.pc ./config/icu-io.pc /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/lib/pkgconfig/
./../src/c-74_2-src-2c60f3f6bd.clean/source/install-sh -c -m 644 ./../src/c-74_2-src-2c60f3f6bd.clean/source/../LICENSE /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/share/icu/icud/74.2/LICENSE
./../src/c-74_2-src-2c60f3f6bd.clean/source/install-sh -c ./config/icu-config /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/../tools/icu/debug/bin/icu-config
./../src/c-74_2-src-2c60f3f6bd.clean/source/install-sh -c -m 644 ./config/Makefile.inc /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/lib/icud/74.2/Makefile.inc
./../src/c-74_2-src-2c60f3f6bd.clean/source/install-sh -c -m 644 ./config/pkgdata.inc /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/lib/icud/74.2/pkgdata.inc
cd /E/kicad-master/build/vcpkg_installed/vcpkg/pkgs/icu_x64-windows/E/kicad-master/build/vcpkg_installed/x64-windows/debug/lib/icud/74.2/..; \
    rm -rf current && ln -s 74.2 current; \
    rm -f Makefile.inc && ln -s current/Makefile.inc Makefile.inc; \
    rm -f pkgdata.inc && ln -s current/pkgdata.inc pkgdata.inc
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "kicad",
  "version": "8.99",
  "license": "GPL-2.0+",
  "dependencies": [
    "boost-algorithm",
    "boost-bimap",
    "boost-filesystem",
    "boost-functional",
    "boost-iterator",
    "boost-locale",
    "boost-optional",
    "boost-property-tree",
    "boost-ptr-container",
    "boost-random",
    "boost-range",
    "boost-test",
    "boost-uuid",
    "cairo",
    "wxwidgets-33",
    "glew",
    "curl",
    {
      "name": "gettext",
      "features": [
        "tools"
      ],
      "host": true
    },
    "harfbuzz",
    "glm",
    {
      "name": "opencascade",
      "features": [
        "rapidjson"
      ]
    },
    "opengl",
    "python3",
    "openssl",
    {
      "name": "sqlite3",
      "features": [
        "fts5",
        "fts4",
        "fts3",
        "rtree",
        "session"
      ]
    },
    "icu",
    "ngspice",
    "wxpython-33",
    {
      "name": "libgit2",
      "features": [
        "ssh",
        "winhttp"
      ]
    },
    "nng",
    "protobuf",
    "zstd"
  ],
  "overrides": [
    {
      "name": "glm",
      "version": "0.9.9.8#2",
      "$comment": "Pinned due to bad vcpkg port updating to a non-real version"
    },
    {
      "name": "python3",
      "version": "3.11.5#0"
    },
    {
      "name": "wxwidgets-33",
      "version": "3.3.0#3"
    },
    {
      "name": "wxpython-33",
      "version": "4.2.3#1"
    },
    {
      "name": "ngspice",
      "version": "44#1"
    },
    {
      "name": "protobuf",
      "version": "3.21.12#4",
      "$comment": "Pinned until compat in kicad is improved"
    }
  ]
}

```
</details>

--- a/ports/icu/fix-using-install-sh.diff
+++ b/ports/icu/fix-using-install-sh.diff
@@ -1,0 +1,23 @@
+diff --git a/source/configure b/source/configure
+index a6df219..2e1ebcf 100644
+--- a/source/configure
++++ b/source/configure
+@@ -4935,12 +4935,12 @@ test -z "$INSTALL_DATA" && INSTALL_DATA='${INSTALL} -m 644'
+ 
+ 
+ 
+-# make sure install is relative to srcdir - if a script
+-if test "$srcdir" = "."; then
+-  # If srcdir isn't just ., then (srcdir) is already prepended.
+-  if test "${ac_install_sh}" = "${INSTALL}"; then
+-   INSTALL="\\\$(top_srcdir)/${ac_install_sh}"
+-  fi
++# If using the included install-sh, make INSTALL work from any build dir.
++if test "${ac_install_sh}" = "${INSTALL}"; then
++    case "$srcdir" in
++        .|./*|../*)  INSTALL="\\\$(top_builddir)/${ac_install_sh}" ;;
++        *) ;;
++    esac
+ fi
+ 
+ # TODO(ICU-20301): Remove fallback to Python 2.

--- a/ports/icu/fix-using-install-sh.diff
+++ b/ports/icu/fix-using-install-sh.diff
@@ -1,5 +1,21 @@
+diff --git a/source/Makefile.in b/source/Makefile.in
+index 9e15b6a016..53330a3921 100644
+--- a/source/Makefile.in
++++ b/source/Makefile.in
+@@ -189,7 +189,10 @@ install-icu: $(INSTALLED_BUILT_FILES)
+ 	$(INSTALL_SCRIPT) $(top_srcdir)/mkinstalldirs $(DESTDIR)$(pkgdatadir)/mkinstalldirs
+ 	$(INSTALL_SCRIPT) $(top_srcdir)/install-sh $(DESTDIR)$(pkgdatadir)/install-sh
+ 	@$(MKINSTALLDIRS) $(DESTDIR)$(libdir)/pkgconfig
+-	$(INSTALL_DATA) $(ALL_PKGCONFIG_FILES) $(DESTDIR)$(libdir)/pkgconfig/
++	# icu's fallback install-sh does not support multi-file install
++	@for file in $(ALL_PKGCONFIG_FILES); do \
++	  $(INSTALL_DATA) "$${file}" $(DESTDIR)$(libdir)/pkgconfig/ ; \
++	done
+ 	$(INSTALL_DATA) $(top_srcdir)/../LICENSE $(DESTDIR)$(pkgdatadir)/LICENSE
+ ifeq ($(INSTALL_ICU_CONFIG),true)
+ 	$(INSTALL_SCRIPT) $(top_builddir)/config/icu-config $(DESTDIR)$(bindir)/icu-config
 diff --git a/source/configure b/source/configure
-index a6df219..2e1ebcf 100644
+index dfa3a091f8..8ee911d42d 100755
 --- a/source/configure
 +++ b/source/configure
 @@ -4935,12 +4935,12 @@ test -z "$INSTALL_DATA" && INSTALL_DATA='${INSTALL} -m 644'

--- a/ports/icu/portfile.cmake
+++ b/ports/icu/portfile.cmake
@@ -12,6 +12,7 @@ vcpkg_extract_source_archive(SOURCE_PATH
         fix_bsd_and_solaris.patch
         fix_parallel_build_on_windows.patch
         fix-python-path-with-spaces.patch
+        fix-using-install-sh.diff
         mh-darwin.patch
         mh-mingw.patch
         mh-msys-msvc.patch

--- a/ports/icu/vcpkg.json
+++ b/ports/icu/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "icu",
   "version": "78.3",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Mature and widely used Unicode and localization library.",
   "homepage": "https://icu.unicode.org/home",
   "license": "ICU",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3942,7 +3942,7 @@
     },
     "icu": {
       "baseline": "78.3",
-      "port-version": 1
+      "port-version": 2
     },
     "ideep": {
       "baseline": "2026-05-11",

--- a/versions/i-/icu.json
+++ b/versions/i-/icu.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "da9f6dddd34300c1eea72aadde1d774ce2d3f31c",
+      "git-tree": "50a263a104075be2c5647af8e723a18914f6ee13",
       "version": "78.3",
       "port-version": 2
     },

--- a/versions/i-/icu.json
+++ b/versions/i-/icu.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "da9f6dddd34300c1eea72aadde1d774ce2d3f31c",
+      "version": "78.3",
+      "port-version": 2
+    },
+    {
       "git-tree": "931a42f3b3b911ba73dbbaead4c1e0037a158efd",
       "version": "78.3",
       "port-version": 1

